.github/workflows/sdk-regression.yml

```diff
@@ -21,19 +21,30 @@ jobs:
           - percy-nightwatch
           - percy-nightmare
           - percy-webdriverio
+          - percy-webdriverio@v2
           - percy-protractor
           - percy-selenium-js
           - percy-selenium-dotnet
           - percy-selenium-java
+          - percy-selenium-python
+          - percy-capybara
+          - percy-appium-js
           - gatsby-plugin-percy
     steps:
+      - uses: jungwinter/split@v2
```

Comment:
let's add a comment on L14, on how to use this, doesn't seem intuitive on first glance.

Reply:
* documentation around not the actions itself, but how are we using it in ourcase, like what does stuff after `@ ` mean to us if someone is adding a new branch they'll read and straight away add that.
* currently reading `percy-webdriverio` and `percy-webdriverio@v2` doesn't sound very intuitive imo.

Reply:
Added doc

Reply:
can you move it to L14 maybe ( basically where we are using it) we're not concerned about the implementation details of the actions